response_types.go

```diff
@@ -0,0 +1,68 @@
+package compute
+
+import (
+	"regexp"
+	"time"
+)
+
+type TokenResponse struct {
+	Token string `json:"token"`
+}
+
+type RunnableResponse struct {
+	Namespace    string `json:"namespace"`
+	FunctionName string `json:"name"`
+	Version      string `json:"version"`
+	Language     string `json:"lang"`
+	DraftVersion string `json:"draftVersion"`
+	APIVersion   string `json:"apiVersion"`
+	FQFN         string `json:"fqfn"`
+}
+
+func (r RunnableResponse) String() string {
+	return r.FQFN
+}
+
+var fqfnRegexp = regexp.MustCompile(`(?P<environment>.*\..*)\.(?P<customer>.*)\#(?P<namespace>.*)::`)
+
+// ToRunnable converts a RunnableResponse type to a Runnable
+func (r RunnableResponse) ToRunnable() *Runnable {
+	match := fqfnRegexp.FindStringSubmatch(r.FQFN)
```

Comment:
There's an `fqfn` package in the Atmo repo that will do this for you without regex, `fqfn.Parse`